.travis.yml

```diff
@@ -23,8 +31,8 @@ install:
 before_script:
   - mkdir build
   - cd build
-  - if [[ "$TRAVIS_OS_NAME" == "linux" ]];then cmake ..; fi;
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then cmake .. -DTINYSPLINE_WITH_PHP=FALSE -DTINYSPLINE_WITH_RUBY=FALSE; fi;
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export CMAKE_FLAGS=$CMAKE_FLAGS -DTINYSPLINE_WITH_PHP=FALSE -DTINYSPLINE_WITH_RUBY=FALSE; fi;
```

Comment:
According to Travis, 
```bash
export CMAKE_FLAGS=$CMAKE_FLAGS -DTINYSPLINE_WITH_PHP=FALSE -DTINYSPLINE_WITH_RUBY=FALSE`
```
fails with exit code 1. I tested this command on linux (due to the lack of an OSX device) and got
```bash
bash: export: `-DTINYSPLINE_WITH_PHP=FALSE': not a valid identifier
bash: export: `-DTINYSPLINE_WITH_RUBY=FALSE': not a valid identifier
```
However, I was able to fix it by quoting the export:
```bash
export CMAKE_FLAGS="$CMAKE_FLAGS -DTINYSPLINE_WITH_PHP=FALSE -DTINYSPLINE_WITH_RUBY=FALSE"
```
This should do the trick for OSX as well.